error.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webpackager
+
+import (
+	"fmt"
+	"net/url"
+)
+// Error represents an error encountered in Packager.Run.
+type Error struct {
+	URL *url.URL
+	Err error
+}
+
+// WrapError wraps err into an Error.
```

Comment:
optional: it's obvious from the impl, but  I believe that still nice to have an explicit description of what the "wrapping" adds.

Reply:
It says an "Error", so the reader is expected to reference the godoc of `Error`. I don't think repeating it is a good idea: it results in duplicate documents, which can get diverted easily. That said, I admit the GoDoc was insufficient, so I added some commnts.